runtime/onert/backend/cpu/ops/LogisticLayer.cc

```diff
@@ -42,8 +42,27 @@ void LogisticLayer::logisticFloat32()
 
 void LogisticLayer::logisticQuant8()
 {
-  // cker quant8 logistic is not implemented yet
-  throw std::runtime_error{"NYI"};
+  uint8_t table[256];
+
+  const auto input_scale = static_cast<double>(_input->data_scale());
+  const auto input_zero_point = static_cast<int32_t>(_input->data_offset());
+  const auto output_scale = static_cast<double>(_output->data_scale());
+  const auto output_zero_point = static_cast<int32_t>(_output->data_offset());
+  const float inverse_scale = 1 / output_scale;
+  int32_t maxval = std::numeric_limits<uint8_t>::max();
+  int32_t minval = std::numeric_limits<uint8_t>::min();
+  for (int32_t val = minval; val <= maxval; ++val)
+  {
+    const float dequantized = input_scale * (val - input_zero_point);
+    const float transformed = 1.0f / (1.0f + std::exp(-dequantized));
+    const float rescaled = std::round(transformed * inverse_scale);
```

Comment:
there are two different way to calculate **quantized**

1. `quantized = static_cast<int32_t>(round(rescaled + output_zero_point)`
2. `quantized = static_cast<int32_t>(rescaled + output_zero_point)`

Is the first method correct?

Reply:
I think that first quantization is more exact than second quantization because of round() and output_zero_point is integer value in my opinion.  I know type casting is round down.
but In some cases, it may be possible to use the second method.
I've referred to tflite.

Reply:
you could port the code from tflite for this implementation. The approach above seems to do most of the computation on de-quantized floats.